File: Arnav Sharma/Assignment-0/if-else/employee.java
import java.util.Scanner;

public class employee {
        public static void main (String[] args) {

            Scanner sc = new Scanner (System.in);


            System.out.println("Enter age");
            int age = sc.nextInt();

            System.out.println("Enter sex: M/F");
            int sex = sc.next().charAt(0);

            System.out.println("Are you married? Y/N");
            int married = sc.next().charAt(0);

            if(sex == 'F') {
                System.out.println("You will work only in urban areas");
            }

            if(sex == 'M') {
                if((age >= 20) && (age < 40)) {
                    System.out.println("You may work anywhere");
                }
                else if((age >= 40) && (age < 60)) {
                    System.out.println("You will work only in urban areas");
                }
                else {
                    System.out.println("ERROR");
                }
                sc.close();

            }
        }
    }

File: sanjana prasad/if-else/employee.cpp
#include <iostream>

using namespace std;

int main()
{
    int age;
    char sex,married;
    cin>>age>>sex>>married;
    if(sex=='F'){
        std::cout << "urban area" << std::endl;
    }
    else if(sex=='M' && (age>20 && age<40)){
        std::cout << "anywhere" << std::endl;
    }
    else if(sex=='M' && (age>=40 && age<60)){
        std::cout << "urban area" << std::endl;
    }
    else std::cout << "ERROR" << std::endl;
    return 0;
}
